array_shift/array_shift.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const shift = module.exports = {};
+
+shift.insertShiftArray = (inputArray, newValue) => {
```

Comment:
"Without utilizing any of the built-in methods available to your language" <- Be careful about this